CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.5.0
+
+* Visualizer can now display predictions alongside the vehicle positions in the ladder view,
```

Comment:
Can you mention the backwards incompatible changes and what needs to be done to upgrade? I think the `opts` now take the `routes` as `%{routes: ...}` rather than directly. And I think calling `new_message` is now `single_file` or something like that?

Reply:
(See 0.2.0 for formatting)

Reply:
ah, yes that is true

Reply:
new message should still ahve the same interface if you want to only send one - in the state, its single_pb, but GTFSRealtimeViz.new_message can still take a single file in the same way as before.